spec/models/bulkrax/importer_spec.rb

```diff
@@ -21,14 +21,20 @@ module Bulkrax
     end
 
     describe 'importer run' do
-      before do
-        allow_any_instance_of(Bulkrax::OaiDcParser).to receive(:collections_total).and_return(1)
-      end
-
-      it 'creates an ImporterRun with total_work_entries set to the value of limit' do
+      it 'finds or creates a new ImporterRun' do
```

Comment:
I think this test is likely wrong now (see comment above)

Reply:
fixed.